dhis-2/dhis-services/dhis-service-analytics/src/test/java/org/hisp/dhis/analytics/event/data/EnrollmentAnalyticsManagerTest.java

```diff
@@ -193,7 +191,7 @@ private void verifyWithProgramStageAndNumericDataElement( ValueType valueType )
 
         if ( valueType == ValueType.NUMBER )
         {
-            subSelect = "coalesce(" + subSelect + ", 'NaN') as fWIAEtYVEGk";
```

Comment:
I'm not sure but I think that returning 'NaN' was a requirement of some client months ago.

Reply:
see comment above